src/OSPSuite.UI/Extensions/ChartControlExtensions.cs

```diff
@@ -36,33 +37,55 @@ public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartF
       /// </summary>
       public static void CopyToClipboard(this UxChartControl chartControl, IChart chart, string watermark)
       {
-         using (var cloneOfChartControl = (ChartControl) chartControl.Clone())
+         using (var cloneOfChartControl = (ChartControl)chartControl.Clone())
          {
             cloneOfChartControl.SetFontAndSizeSettings(chart.FontAndSize, chartControl.Size);
 
             if (chart.IncludeOriginData)
                AddOriginData(cloneOfChartControl, chart);
 
-            AddWatermark(cloneOfChartControl, watermark, chart);
+            prepareChartForCopying(cloneOfChartControl);
 
-            chartControl.CopyChartToClipboard(cloneOfChartControl);
+            if (!string.IsNullOrEmpty(watermark))
+            {
+               //Bug in devexpress that does not keep the color in bar charts
```

Comment:
Do we agree that this comment is at the wrong place here as we have a chart of type IChart? Please move at the right place